### PR TITLE
deleting all tasks

### PR DESCRIPTION
**Pull request by Sandrine21**
#### What does this PR do?
This PR is indicating delete all tasks

#### Description of Task to be completed?
this tasks allow user to delete all tasks, she has entered. she will be able to view tasks and delete them all.

#### How should this be manually tested?
Clone it
run Flutter pub get to install dependencies.
run 'flutter run' to start.

#### Any background context you want to provide?
make sure to first enter data so that you can be able to view those data and delete them all.

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)

Screenshot_20201112-112035
Screenshot_20201112-112039
Screenshot_20201112-112044

Questions:

### CODE REVIEW

**Comment by Sandrine21** [comment]
User will be able to delete all task.

File: lib/View-tasks.Dart [hunk 1]
```diff
@@ -11,69 +12,67 @@ class ViewScreen extends StatefulWidget {
...
+      ),
+      body:
+          widget.mytask.isEmpty 
+              ? Center(
+                  child: Text(
+                    'No tasks added yet',
+                    style: TextStyle(
+                      fontSize: 30.0,
+                    ),
+                    // textAlign: TextAlign.center,
```
Comment (mujohn26) [inline lib/View-tasks.Dart]: @Sandrine21 Please remove these comments

**Review by mujohn26 — CHANGES_REQUESTED** [review]

**Comment by mujohn26** [comment]
> _No description provided._

@Sandrine21 Please I need descriptions of this PR, I really don't know what you worked on.

**Review by mujohn26 — APPROVED** [review]
Good !

**Pull request merged**